jep/9999/README.adoc

```diff
@@ -0,0 +1,134 @@
+* Is the plugin more than just superficial themeing/UI? While those have their place and are popular, advertising plugins changing the UI out of the box would be very weird (Simple Theme Plugin, Green Balls)
+* Are the features it provides useful to someone fairly new to Jenkins? If the plugin requires someone to e.g. be familiar with Jenkins internals before they can use it successfully, or to have multiple Jenkins instances, it may not be a good choice for this dialog. For example, if the script console were a plugin, it would probably not be on here.
+* Even if it's not popular, is it a reasonably common SCM? SCMs are so fundamental to successful Jenkins use that these shouldn't be as heavily restricted as some of the other categories.
+* The plugin most work without requiring a restart of Jenkins.
+* The plugin must be discoverable when installed, and at least allow users to get started without outside documentation. (This disqualifies Build Token Root Plugin.)
+* The plugin is hosted in the link:https://github.com/jenkinsci[jenkinsci] GitHub organization
+* The plugin must be maintained reasonably well (e.g. was updated in the last 18 months or so, and ideally has a dedicated maintainer)
+
+=== Categories
+
+Given the diverse feature set of Jenkins plugins, it is difficult to define a
+set of categories that can be used to group plugins without misrepresenting
+them (and having every plugin only how up once in the list, rather than
+multiple times). The following categories appear to adequately provide a
+high-level view of plugin functionality in Jenkins:
+
+* **General**: Anything that has no better category. Depending on contents, these may result in the creation of new categories. Possible examples are Rebuilder Plugin, Timestamper Plugin, or Translation Assistance Plugin.
+* **Organization and Administration**: Plugins that help Jenkins administrators accomplish their goals, and that help keep order. Examples are Monitoring Plugin, Antisamy Markup Formatter, or Disk Usage Plugin.
+* **Build Features** are plugins that provide features to builds/jobs typically beyond an SCM, build step or publisher. SSH Agent Plugin, AnsiColor Plugin, and Timestamper.
+* **Build Tools**: for plugins providing build tool integration, e.g. NodeJS Plugin, Maven Plugin, Gradle Plugin.
+* **Build Analysis and Reporting**: Plugins in this category process reports or data generated during builds, such as JUnit Plugin, Build Failure Analyzer, or Checkstyle Plugin.
+* **Build Pipelines and Continuous Delivery**: is for plugins that allow creating more complex chains of actions that would reasonably be possible in a single freestyle job. Examples are Copy Artifact Plugin, Parameterized Trigger Plugin, or Workflow Plugin.
+* **SCM**:provides a selection of plugins that integrate with SCM tools, e.g. Subversion Plugin, GitHub Plugin, or GitHub Pull Request Builder Plugin.
```

Comment:
`github` is a grab bag of features actually, not an SCM plugin per se. `github-branch-source` might be a better example.